pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -143,7 +145,7 @@ func (p RESTStorageProvider) PostStartHook() (string, genericapiserver.PostStart
 
 // Returns false if there's a "exempt" priority-level existing in the cluster, otherwise returns a true
 // if the "exempt" priority-level is not found.
-func lastMandatoryExists(flowcontrolClientSet flowcontrolclient.FlowcontrolV1alpha1Interface) (bool, error) {
+func lastMandatoryExists(flowcontrolClientSet flowcontrolclient.FlowcontrolV1beta1Interface) (bool, error) {
```

Comment:
BTW, there is an old problem here: the name of this function is the opposite of what it means.  Put another way, the return value from this function is the opposite of what the name of this function says.